crates/ln-dlc-node/src/tests/just_in_time_channel/offline_receiver.rs

```diff
@@ -0,0 +1,49 @@
+use crate::ln::JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT;
+use crate::node::Node;
+use crate::tests::init_tracing;
+use crate::tests::just_in_time_channel::create::send_interceptable_payment;
+use crate::tests::just_in_time_channel::TestPath;
+use crate::tests::min_outbound_liquidity_channel_creator;
+use bitcoin::Amount;
+
+#[tokio::test]
+#[ignore]
+async fn offline_receiver() {
```

Comment:
could we come up with a better name for this test?
`able_to_receive_an_offline_payment`? or something? it would be immensely helpful to have a name that's obvious when it fails.

Reply:
The nameing could really be better, but it's also not really an offline payment. I am going to merge it as is and we can change the name in a follow up PR.